docs/compatibility.md

```diff
@@ -0,0 +1,8 @@
+## Compatibility 
```

Comment:
Does it make sense to put this into the README.md to have it visible all the time?